packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -356,15 +378,29 @@ export const GridRootStyles = styled('div', {
         },
       },
     },
+    [`& .${gridClasses['container--top']}, & .${gridClasses['container--bottom']}`]: {
+      '[role=row]': {
+        background: 'var(--unstable_DataGrid-containerBackground)',
+      },
+      [`.${gridClasses.pinnedColumnHeaders} [role=row]`]: {
+        background: 'var(--unstable_DataGrid-pinnedBackground)',
+      },
+    },
     [`& .${gridClasses.cell}`]: {
       display: 'flex',
       alignItems: 'center',
+      '--width': '0px',
+      '--height': '0px',
+      minWidth: 'var(--width)',
+      maxWidth: 'var(--width)',
+      minHeight: 'var(--height)',
+      maxHeight: 'var(--height)',
```

Comment:
I never understood why Damien didn't simply do

```suggestion
      width: 'var(--width)',
      height: 'var(--height)',
```

Reply:
Done